bundle/Form/Type/FieldType/UserType.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\InformationCollectionBundle\Form\Type\FieldType;
+
+use Ibexa\Contracts\Core\Repository\Values\ContentType\FieldDefinition;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Validator\Constraints;
+
+abstract class UserType extends AbstractType
```

Comment:
We won't need this for info collection, please remove it.

Reply:
solved in 1ab8f5d6